configure.py

```diff
@@ -187,7 +187,7 @@
 parser.add_option('--experimental-http-parser',
     action='store_true',
     dest='experimental_http_parser',
-    help='use llhttp instead of http_parser by default')
+    help='(no-op)')
```

Comment:
If we are going to land this patch fairly soon (before April), it's probably easier if we simply flip the default value of the build time switch but do not actually remove it, and postpone the actual removal of the code when we have more usage in the wild.

Reply:
@joyeecheung The thing is … is there a way to flip the default value here? How do we do that? I assume we could introduce a new flag `--no-experimental-http-parser` or so, but that seems … idk?

Reply:
Does adding `default=True` here work? (From what I can tell it should do the trick?)

Also I just notice that we have a bunch of '--shared-http-parser*` flags below, should we change the help text of them as well? (They should be practically turned into noops by this patch since the library is not actually used?)

Reply:
@joyeecheung That would leave us with no way to turn it off, right? So it’s the same as making this a no-op?